packages/go_router_builder/example/pubspec.yaml

```diff
@@ -18,15 +17,8 @@ dev_dependencies:
   build_verify: ^3.0.0
   flutter_test:
     sdk: flutter
-  # Uses path dependency override
-  go_router_builder:
+  go_router_builder: ^1.0.0
```

Comment:
This can still be path-based (and should). It's only cross-package references that are a problem.